rules/defense_evasion_process_execution_from_self_deleting_binary.yml

```diff
@@ -19,30 +19,15 @@ references:
 
 condition: >
   sequence
-  maxspan 1m 
-    |delete_file and file.info.is_disposition_delete_file
+    maxspan 1m
+    |delete_file
       and
-      not
-     ps.exe imatches 
-      (
-        '?:\\$WINDOWS.~BT\\Sources\\SetupHost.exe',
-        '?:\\WINDOWS\\uus\\packages\\preview\\*\\wuaucltcore.exe'
-      )
-      and 
-      not
-     ps.cmdline imatches 
-      (
-        '?:\\Windows\\system32\\svchost.exe -k wsappx -p -s AppXSvc'
-      )
+    file.info.is_disposition_delete_file
       and
-      not 
-     file.path imatches 
-      (
-        '?:\\Windows\\SoftwareDistribution\\Download\\*',
-        '?:\\Windows\\uus\\packages\\preview\\*'
-      )
+    not file.name imatches '?:\\Windows\\SoftwareDistribution\\Download\\*'
```

Comment:
@N0vaSky The `?:\\Windows\\SoftwareDistribution\\Download\\*` exclusion already exists in the upstream rule. We want to keep the remaining exclusions as well, so I think we can drop this change altogether.